src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -119,7 +119,7 @@ trait NodeStatusBuilder {
   lazy val nodeStatusHolder = Agent(nodeStatus)
 }
 
-trait BlockChainBuilder {
+trait BlockchainBuilder {
```

Comment:
Super Minor: I like naming standardization so probably we will need to change other Block**C**hain names, for example:

- `io.iohk.ethereum.ets.blockchain.BlockChainTestConfig`
- `io.iohk.ethereum.blockchain.sync.FastSync.SyncingHandler#assignBlockChainWork`

WDYT?